landmarkerio/lmio

```diff
@@ -76,6 +76,9 @@ if __name__ == "__main__":
                         help="Listen to public requests (0.0.0.0).")
     parser.add_argument("-p", "--port",
                         help="The port to host the server on. 5000 by default")
+    parser.add_argument('--rewrite_templates', action='store_true',
```

Comment:
can we change this to `--upgrade-templates`? Prefer avoiding `_` for CLI opts, and upgrade makes it clear that yml is the future